src/data/device_registry.ts

```diff
@@ -31,7 +31,7 @@ export const updateDeviceRegistryEntry = (
     ...updates,
   });
 
-const fetchDeviceRegistry = (conn) =>
+export const fetchDeviceRegistry = (conn) =>
```

Comment:
Don't use this, use `subscribeDeviceRegistry` instead. Keeps you up to date.

Reply:
OK, should the returned unsubscribe callback be called from somewhere?

Reply:
inside `disconnectedCallback`

Reply:
https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/panels/config/integrations/ha-config-integrations.ts#L64-L81

Reply:
Thanks, fixed.